apps/explorer/src/components/ownedobjects/views/ObjectFieldsCard.tsx

```diff
@@ -0,0 +1,249 @@
+                                                            >
+                                                                {typeof value}
+                                                            </Text>
+                                                        </div>
+                                                    </div>
+                                                </ListItem>
+                                            </div>
+                                        ))}
+                                    </VerticalList>
+                                </div>
+                            </div>
+
+                            <div className="grow overflow-auto border-gray-45 pt-1 md:w-3/5 md:border-l md:pl-7">
+                                <div className="flex max-h-[600px] flex-col gap-5 overflow-x-scroll pb-5">
+                                    {Object.entries(fieldsData).map(
+                                        ([key, value]) => (
+                                            <div key={key}>
+                                                <DisclosureBox
+                                                    title={
+                                                        <Text
+                                                            variant="body/medium"
+                                                            color="steel-dark"
+                                                        >
+                                                            {key.toString()}:
+                                                        </Text>
+                                                    }
+                                                    preview={
+                                                        <div className="flex items-end gap-1 truncate break-all">
+                                                            {typeof value ===
+                                                            'object' ? (
+                                                                <Text
+                                                                    variant="body/medium"
+                                                                    color="gray-90"
+                                                                >
+                                                                    Click to
+                                                                    view
+                                                                </Text>
+                                                            ) : (
+                                                                <FieldItem
+                                                                    value={
+                                                                        value
+                                                                    }
+                                                                    type={key}
```

Comment:
Seems weird that the key is the type here? I think using the normalized module would help here